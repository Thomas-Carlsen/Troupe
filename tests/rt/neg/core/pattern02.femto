let val 10 = 20 in () end 
