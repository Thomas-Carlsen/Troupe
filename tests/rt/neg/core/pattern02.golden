2019-03-27T19:47:44.005Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
[31mRuntime error in thread 5649b18d-7337-4748-9f88-c9e0c6dc3c77@{}%{}[39m
[31m>> pattern match failure in let declaration at :1:5[39m
